src/frontend/komponenter/Fagsak/Saksoversikt/Behandlinger.tsx

```diff
@@ -10,16 +10,40 @@ import type { IMinimalFagsak } from '../../../typer/fagsak';
 import type { ITilbakekrevingsbehandling } from '../../../typer/tilbakekrevingsbehandling';
 import { kalenderDiff } from '../../../utils/kalender';
 import { Behandling } from './Behandling';
+import { BehandlingEllerTilbakebetaling } from './BehandlingEllerTilbakebetaling';
 import type { VisningBehandling } from './visningBehandling';
 
 interface IBehandlingshistorikkProps {
     minimalFagsak: IMinimalFagsak;
 }
 
+function konverterBehandling(visningBehandling: VisningBehandling): BehandlingTabellobjekt {
+    return {
+        behandlingEllerTilbakemelding: BehandlingEllerTilbakebetaling.BEHANDLING,
+        faktiskObjekt: visningBehandling,
+    };
+}
+
+function konverterTilbakekrevingsbehandling(
+    tilbakekrevingsbehandling: ITilbakekrevingsbehandling
+): BehandlingTabellobjekt {
+    return {
+        behandlingEllerTilbakemelding: BehandlingEllerTilbakebetaling.TIlBAKEBETALING,
+        faktiskObjekt: tilbakekrevingsbehandling,
+    };
+}
+
+interface BehandlingTabellobjekt {
+    behandlingEllerTilbakemelding: BehandlingEllerTilbakebetaling;
+    faktiskObjekt: VisningBehandling | ITilbakekrevingsbehandling;
+}
```

Comment:
Hvorfor trenger vi dette interfacet? Ser ikke at  `behandlingEllerTilbakemelding` blir brukt noe sted, så skjønner ikke helt hva vi tjener på å ha det feltet.

Reply:
Skulle gjerne ha sloppe det, men i visRad-metoden her bruker vi det:
`const visRad = (behandling: BehandlingTabellobjekt, visHenlagteBehandlinger: boolean) => {
    if (visHenlagteBehandlinger) return true;
    if (!behandling.faktiskObjekt.resultat) return false;
    **if (behandling.behandlingEllerTilbakemelding === BehandlingEllerTilbakebetaling.BEHANDLING) {**
        return !erBehandlingHenlagt(behandling.faktiskObjekt.resultat as BehandlingResultat);
    }
    return Behandlingsresultatstype.HENLAGT !== behandling.faktiskObjekt.resultat;
};`

Synes ikkje det vart veldig pent, men eg fann ikkje nokon betre måte å gjera det på. Problematikken skuldast at behandling-inputen her er _VisningBehandling | ITiITilbakekrevingsbehandling_, og at desse har kvar sin type for resultat-objektet. Dermed er resultat _Behandlingsresultatstype | BehandlingResultat | undefined_, og da var det vanskeleg å få kalt _erBehandlingHenlagt_-metoden, som kun tar BehandlingResultat.

Reply:
Sett. Da gir dette mening ja